pages/index.tsx

```diff
@@ -42,4 +44,33 @@ export default function Home() {
       </footer>
     </div>
   );
-}
+};
+
+export default Home;
+
+const containerStyle = css({
+  boxSizing: 'border-box',
+  backgroundColor: '#0066ff',
+  color: '#fff',
+  textAlign: 'center'
+});
+
+type StyledLinkProps = {
```

Comment:
[non-blocking] this might be worth splitting out into another component, seems intuitively useful to globalise the `Link`s in your app

Reply:
Makes sense. I thought about doing this later on when I actually wanted to re-use it, but I guess it makes more sense to do it now. Done.